tests/files/aspace_metadata_delimited.csv

```diff
@@ -0,0 +1,3 @@
+uri,title,file_identifier,authors
```

Comment:
One small thing: I would rename this folder to "fixtures" instead of "files", as that will make it clear that's what these are for.